src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentClient.kt

```diff
@@ -226,4 +251,16 @@ class CodyAgentClient(private val project: Project, private val webview: NativeW
 
     return saveFileFuture
   }
+
+  @JsonNotification("window/didChangeContext")
+  fun window_didChangeContext(params: Window_DidChangeContextParams) {
+    println(params)
```

Comment:
Let's not spam the log.

Is this, in fact, important for refreshing the status bar icon? `"cody.activated"` will be `true` when the extension thinks it is good to go.

Reply:
Yeah, refreshing status icon bar is part I/ know I missed but I forget to add it at the end.
Thanks, I will add it now!